### ISSUE TITLE
add splitSum option 

### ISSUE THREAD

**Issue by azazellochg** [scipion-em/scipion-em-motioncorr#40]
to generate odd/even sums for CryoCARE.

**Comment by JorMaister** [comment]
Hi @azazellochg ,
We're working on the correction of the tilt series movies with xmipp-flexalign, but I'll do it in parallel, so we can offer more than one option for this operation.

**Comment by azazellochg** [comment]
Hi Jorge,

Thank you! I'll have a look at the tomo branch if I can this WE.

Best regards,
Grigory

--------------------------------------------------------------------------------
Grigory Sharov, Ph.D.

MRC Laboratory of Molecular Biology,
Francis Crick Avenue,
Cambridge Biomedical Campus,
Cambridge CB2 0QH, UK.
tel. +44 (0) 1223 267228 <+44%201223%20267228>
e-mail: gsharov@mrc-lmb.cam.ac.uk

**Comment by azazellochg** [comment]
If I understand correctly, splitEvenOdd option exists for ProtTsCorrectMotion base protocol but I dont know what programs use this class at the moment.

**Comment by JorMaister** [comment]
Hi @azazellochg!
For now, cryoCARE is the only one using it (its data preparation via FlexAlign for the motion correction), but it was considered in the lab that it should be useful to have that even/odd frames splitting functionality implemented.

**Comment by azazellochg** [comment]
Hi @JorMaister ,
so in principle it would be nice to have a separate mc2 protocol inheriting from ProtTsCorrectMotion class, right?

**Comment by delarosatrevin** [comment]
I think this is the strategy that we have used in similar cases. We have created a separate protocol for tomography even if the use the same program under the hood (e.g CTFfind)

**Comment by JorMaister** [comment]
Ok, I'll implement it that way, as a new protocol inside motioncor2 plugin. 
Thanks, @azazellochg and @delarosatrevin

**Comment by azazellochg** [comment]
Done in 2941bee51044fd8d10a82503253f7d42a2fafd18
